scanner/src/components/Header/index.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import LeftArrowSvg from "$assets/left-arrow.svg"
+  import QuestionSvg from "$assets/question.svg"
+  import MetaDataPng from "$assets/metadata.png"
+</script>
+
+<div class="wrapper">
+  <a href="/">
+    <img class="left-arrow" src={LeftArrowSvg} alt="" aria-hidden="true" />
+  </a>
```

Comment:
Is `/` where we scan codes?

Reply:
Its the start page with `Turn On` button

Reply:
Exactly, why would we want to go back to that page? It should go back, to the previous page, which is the `/scan` route as far as I know